src/components/Payment/Payment.tsx

```diff
@@ -170,7 +170,8 @@ const Payment = ({
       {accessModel === ACCESS_MODEL.SVOD && (
         <div className={panelClassName}>
           <div className={panelHeaderClassName}>
-            <h3>{isChangingOffer ? t('user:payment.change_plan') : t('user:payment.subscription_details')}</h3>
+            {!isChangingOffer && <h1 className="hideUntilFocus">{t('nav.payments')}</h1>}
```

Comment:
Why is this conditional? We always need the hidden h1 right?

Reply:
Your feedback seems logical Roy, but I have to see the whole context before i can make that assumption. Good call, though.

Reply:
Yes we do! Just like I explained in a previous comment:
In some occasions, depending if you're on a SVOD or AUTHVOD config and depending of the value of the variable `isChangingOffer`, one of the Payment element is shown or not. I've used that element to determine where the hidden `<h1>` element should be.